OSDOCS-14525 added files for the installation section (#93011)
openshift/openshift-docs: modules/zero-trust-manager-uninstall-resources.adoc

// Module included in the following assemblies:
//
// * security/zero_trust_workload_identity_manager/zero-trust-manager-uninstall.adoc

:_mod-docs-content-type: PROCEDURE
[id="zero-trust-manager-uninstall-resources-console_{context}"]
= Uninstalling {zero-trust-full} resources by using the CLI

> Once you have uninstalled the {zero-trust-full}, you have the option to delete its associated resources from your cluster.

Reviewer: ```suggestion
After you have uninstalled the {zero-trust-full}, you can delete its associated resources from your cluster.
```

$ oc delete service -l=app.kubernetes.io/managed-by=zero-trust-workload-identity-manager
----

. Uninstall the namespace by running the following command:
+
[source,terminal]
----
$ oc delete ns zero-trust-workload-identity-manager
----

. Uninstall the cluster-wide role-based access control (RBAC) by running each of the following commands:
+
[source,terminal]
----
$ oc delete clusterrolebinding -l=app.kubernetes.io/managed-by=zero-trust-workload-identity-manager
$ oc delete clusterrole -l=app.kubernetes.io/managed-by=zero-trust-workload-identity-manager
----

. Uninstall the Custom Resource Definitions (CRDs) by running each of the following commands:
+
[source,terminal]
----
$ oc delete crd spireserverconfigs.operator.openshift.io
$ oc delete crd spireagentconfigs.operator.openshift.io
$ oc delete crd spiffecsidriverconfigs.operator.openshift.io
$ oc delete crd spireoidcdiscoveryproviderconfigs.operator.openshift.io
$ oc delete crd clusterfederatedtrustdomains.spire.spiffe.io
$ oc delete crd clusterspiffeids.spire.spiffe.io
$ oc delete crd clusterstaticentries.spire.spiffe.io
----

.Verification

> If you want to verify that the resources have been deleted, replace each `oc delete` command with `oc get` and run the command. If no resources are returned, the deletion was successful.

Reviewer: ```suggestion
To verify that the resources have been deleted, replace each `oc delete` command with `oc get`, and then run the command. If no resources are returned, the deletion was successful.
```
